mkdocs/commands/gh_deploy.py

```diff
@@ -14,8 +14,12 @@
 
 
 def _is_cwd_git_repo():
-    proc = subprocess.Popen(['git', 'rev-parse', '--is-inside-work-tree'],
+    try:
+        proc = subprocess.Popen(['git', 'rev-parse', '--is-inside-work-tree'],
                             stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    except FileNotFoundError:
+        log.error("Could not find git - is it installed / on path?")
```

Comment:
I understand the purpose of the slash in the error message, but it could be confusing to those who actually need it. Perhaps we should reword the message. A few options:

* "... installed properly"
* "... installed and on your path"

Also, a note should be added to the release notes.

With those two changes, I'll merge this.